kybra/__main__.py

```diff
@@ -20,6 +20,15 @@
 
 @timed
 def main():
+    if sys.argv[1] == "install-dfx-extension":
```

Comment:
Should I comment that this is temporary? Or explain that a more official way to install is coming?

Reply:
Comment added